### PR TITLE
JA translation Is it ready yet

### PR DESCRIPTION
**Pull request by mattletford**
Convert .docx to Markdown for JA translation of topics api experiment and participate @agektmr

### CODE REVIEW

**Comment by agektmr** [comment]
How is this different from #2650 ?

**Comment by agektmr** [comment]
The file must be located at `site/ja/docs/privacy-sandbox/status/index.md` site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md`.

File: site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 実装ステータス
+date: 2021-05-18
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md:4]:
```suggestion
subhead: Privacy Sandbox API の実装ステータス。注意 API ごとに個別のオリジン トライアル期間が複数ある場合があります。
description: Privacy Sandbox API の実装ステータス。注意 API ごとに個別のオリジン トライアル期間が複数ある場合があります。
date: 2021-05-18
```

File: site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+updated: 2022-03-31
+authors:
+  - samdutton
+  - alexandrawhite
+---
+
+[実装ステータス](/docs/privacy-sandbox/status/)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md:11]: Remove this line.
```suggestion
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by mattletford** [comment]
I've applied the suggestions @agektmr

**Comment by apebase** [comment]
> 

Thank you, there is already an index.md file located at site/ja/docs/privacy-sandbox/status/index.md, titled '準備はできていますか？'

**Comment by agektmr** [comment]
@apebase This is an update. Please overwrite it.

**Comment by mattletford** [comment]
> @apebase This is an update. Please overwrite it.

@agektmr I've overwritten the file

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 3]
```diff
@@ -1,99 +1,165 @@
...
 authors:
   - samdutton
+  - alexandrawhite
 ---
 
-{% Aside 'caution' %}APIごとに複数の個別のオリジントライアル期間が存在する場合があります。 {% endAside %}
+Privacy Sandbox API の実装ステータス。注意 API ごとに個別のオリジン トライアル期間が複数ある場合があります。
 
-## アトリビューションレポート
+## [Attribution Reporting](/docs/privacy-sandbox/status/#attribution-reporting)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:15]:
```suggestion
## [アトリビューション レポート](/docs/privacy-sandbox/status/#attribution-reporting)
```

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 4]
```diff
@@ -1,99 +1,165 @@
...
-- [GitHub](https://github.com/WICG/conversion-measurement-api/) ：APIの質問とディスカッションについては[問題](https://github.com/WICG/conversion-measurement-api/issues)を参照してください。
 
-### 状況：詳細
+- [Chrome プラットフォームのステータス](https://www.chromestatus.com/features/6412002824028160)。
 
-[状況](/docs/privacy-sandbox/attribution-reporting-introduction/#status)を参照してください。
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=attribution%20reporting)。
 
-### すべてのリソース
+- [GitHub](https://github.com/WICG/conversion-measurement-api/): API に関する質問とディスカッションについては、[問題](https://github.com/WICG/conversion-measurement-api/issues)をご覧ください。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:25]:
```suggestion
- [GitHub](https://github.com/WICG/conversion-measurement-api/): API に関する質問とディスカッションについては、[イシュー](https://github.com/WICG/conversion-measurement-api/issues)をご覧ください。
```

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 5]
```diff
@@ -1,99 +1,165 @@
...
-## Trust Tokens
+[ステータス](/docs/privacy-sandbox/attribution-reporting/#status)をご覧ください。
+
+### [すべてのリソース](/docs/privacy-sandbox/status/#all-resources)
+
+- [Attribution Reporting（Conversion Measurement](/docs/privacy-sandbox/attribution-reporting/)
+
+- [Attribution Reporting（Conversion Measurement）の概要](/docs/privacy-sandbox/attribution-reporting-introduction/)
+
+- [API の技術解説](https://github.com/WICG/conversion-measurement-api/)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:37]: - [API のテクニカル Explainer](https://github.com/WICG/conversion-measurement-api/)

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 6]
```diff
@@ -1,99 +1,165 @@
...
-[TURTLEDOVEの](https://github.com/WICG/turtledove)から生まれたAPI。
+- 最初の[オリジントライアル](https://web.dev/origin-trials)は終了しました。最新情報については、[Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/MmijXrmwrJs) をご覧ください。
+
+- [初期バージョンのデモ](https://floc.glitch.me/)（オリジン トライアルは終了しました）。
+
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=floc)。
+
+- [API の提案](https://github.com/WICG/floc)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/WICG/floc): API に関する質問とディスカッションについては、[問題](https://github.com/WICG/floc/issues)をご覧ください。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:105]:
```suggestion
- [GitHub](https://github.com/WICG/floc): API に関する質問とディスカッションについては、[イシュー](https://github.com/WICG/floc/issues)をご覧ください。
```

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 7]
```diff
@@ -1,99 +1,165 @@
...
-- [APIの提案](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)は、 [WICG](https://www.w3.org/community/wicg/)および利害関係者グループと議論中です。
-- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) ：APIの質問とディスカッションについて[は、TURTLEDOVEの問題](https://github.com/WICG/turtledove/issues)を参照してください。
 
-<br>
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=fledge)。
 
----
+- [API の提案](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) : API に関する質問とディスカッションについては、[TURTLEDOVE の問題](https://github.com/WICG/turtledove/issues)をご覧ください。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:123]:
```suggestion
- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) : API に関する質問とディスカッションについては、[TURTLEDOVE のイシュー](https://github.com/WICG/turtledove/issues)をご覧ください。
- ```

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 8]
```diff
@@ -1,99 +1,165 @@
...
+
+- [Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/oTwd6VwCwqs)。
+
+- [Intent to Prototype](https://groups.google.com/a/chromium.org/g/blink-dev/c/59uTw_dxM3M/m/vF9lF9BVAgAJ)。
+
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=%22topics%20api%22)。
+
+- [API の提案](https://github.com/jkarlin/topics)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/jkarlin/topics/blob/main/README.md) : API に関する質問とディスカッションについては、[Topics API の問題](https://github.com/jkarlin/topics/issues)をご覧ください。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:137]:
```suggestion
- [GitHub](https://github.com/jkarlin/topics/blob/main/README.md) : API に関する質問とディスカッションについては、[Topics API のイシュー](https://github.com/jkarlin/topics/issues)をご覧ください。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

File: site/ja/docs/privacy-sandbox/status/index.md (hunk 5 above)
Comment (mattletford) [reply site/ja/docs/privacy-sandbox/status/index.md]: @agektmr I've committed the changes and updated the link

**Review by agektmr — APPROVED** [review]
LGTM

File: site/ja/docs/privacy-sandbox/status/index.md [hunk 9]
```diff
@@ -1,99 +1,168 @@
...
-- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) ：APIの質問とディスカッションについて[は、TURTLEDOVEの問題](https://github.com/WICG/turtledove/issues)を参照してください。
 
-<br>
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=fledge)。
 
----
+- [API の提案](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) : API に関する質問とディスカッションについては、[TURTLEDOVE のイシュー](https://github.com/WICG/turtledove/issues)をご覧ください。
+- ```
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/status/index.md:124]: Please remove this empty code block.

Comment (mattletford) [reply site/ja/docs/privacy-sandbox/status/index.md]: @agektmr I've removed the empty code block

**Review by agektmr — APPROVED** [review]
LGTM

File: site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md (hunk 1 above)
Comment (S3od11) [reply site/ja/docs/privacy-sandbox/is-it-ready-yet/index.md]: ً

**Pull request merged**